server/app/routes/api/reviews.js

```diff
@@ -1,20 +1,69 @@
 var express = require('express');
-var router = express.Router();
+var router = express.Router({ mergeParams: true });
 var mongoose = require('mongoose');
 
+require('../../../db/models/review');
+var Review = mongoose.model('Review');
+
+
 // available to eveyone
-router.get('/:id/reviews', function (req, res) {
+router.get('/', function (req, res, next) {
   var id = req.params.id;
-  mongoose.model('User')
-  .findById(id)
-  .then(function(user) {
-    return user.findReviews()
-  })
+  Review.find({author: id})
   .then(function(reviews) {
     res.json(reviews)
   })
-  .then(null, function(err) {
-    console.log(err);
-    res.sendStatus(404);
+  .then(null, next)
+})
+
+router.post('/', function(req, res, next) {
```

Comment:
A user would be posting a review for a specific wine. How does this route ensure that the review is associated with a wine?